roles/openshift_descheduler/templates/descheduler-cronjob.yaml.j2

```diff
@@ -21,6 +21,9 @@ spec:
             - /bin/descheduler
             args:
             - --policy-config-file=/policy-dir/policy.yaml
+{% if openshift_descheduler_debug_level is not none %}
+            - --v={{ openshift_descheduler_debug_level }}
```

Comment:
Since there will always be a value for openshift_descheduler_debug_level due to the default, lets get rid of the conditional logic here.